src/main/java/com/facebook/openwifirrm/optimizers/tpc/MeasurementBasedApApTPC.java

```diff
@@ -326,21 +325,24 @@ protected static int computeTxPower(
 	/**
 	 * Calculate new tx powers for the given band.
```

Comment:
can we update this since now it's taking in a channel? I know that channel maps to band but would be good to be explicit in the doc